src/PotentialCalculation/PotentialCalculationSetup/PotentialCalculationSetupCylindrical.jl

```diff
@@ -81,7 +81,8 @@ end
 
 function PotentialCalculationSetup(det::SolidStateDetector{T}, grid::CylindricalGrid{T}, 
                 medium::NamedTuple = material_properties[materials["vacuum"]],
-                potential_array::Union{Missing, Array{T, 3}} = missing; 
+                potential_array::Union{Missing, Array{T, 3}} = missing,
+                imp_scale::Union{Missing, ElectricPotential{T, 3}} = missing; 

```

Comment:
Is there a reason not to pass `imp_scale` as `Array{T,3}` like you do for the `potential_array`.
It looks like you only access the `data` field and I guess the `grid` is the same as in `point_types`..

Reply:
Yes, I think we actually do not need the `point_types` anymore (in that function).
We can just multiple all elements of `ϵ` with `(1 - imp_scale)` and do not need to check 
if it is marked as an undepleted point type.